### PR TITLE
New FR systems

### PR DESCRIPTION
**Pull request by sthocs**
Hi, thank you for working on sush a standard!
I'd like to add 3 new FR Smoove systems:
- Saint-Etienne
- Vannes
- Avignon

I found them listed on https://www.data.gouv.fr/fr/search/?q=gbfs

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -65,6 +65,9 @@ ES,Bilbaobizi (Bilbao),"Bilbao, ES",nextbike_bo,https://www.bilbaobizi.bilbao.eu
 ES,Sitycleta (Las Palmas),"Las Palmas de Gran Canaria, ES",nextbike_el,https://www.sitycleta.com/xx/laspalmas/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_el/gbfs.json
 FR,LE vélo STAR,"Rennes, FR",le_velo_star,https://www.star.fr/le-velo/nos-offres/vls/,https://eu.ftp.opendatasoft.com/star/gbfs/gbfs.json
 FR,Libélo,"Valence, FR",libelo,http://www.citea.info/presentation/?rub_code=92,https://valence.publicbikesystem.net/ube/gbfs/v1/gbfs.json
+FR,Vélivert,"Saint-Etienne, FR",Velivert_FR_Saint-Etienne,https://www.velivert.fr/,https://saint-etienne-gbfs.klervi.net/gbfs/gbfs.json
```
Comment (bddq) [inline systems.csv:68]: I suggest IDs for Smoove networks to start with `smoove`.
```suggestion
FR,Vélivert,"Saint-Etienne, FR",smoove_velivert,https://www.velivert.fr/,https://saint-etienne-gbfs.klervi.net/gbfs/gbfs.json
```

File: systems.csv [hunk 2]
```diff
@@ -65,6 +65,9 @@ ES,Bilbaobizi (Bilbao),"Bilbao, ES",nextbike_bo,https://www.bilbaobizi.bilbao.eu
...
+FR,Vélocéo,"Vannes, FR",Veloceo_FR_Vannes,https://veloceo.kiceo.fr/,https://vannes-gbfs.klervi.net/gbfs/gbfs.json
```
Comment (bddq) [inline systems.csv:69]: I suggest IDs for Smoove networks to start with `smoove`.
```suggestion
FR,Vélocéo,"Vannes, FR",smoove_veloceo,https://veloceo.kiceo.fr/,https://vannes-gbfs.klervi.net/gbfs/gbfs.json
```

File: systems.csv [hunk 3]
```diff
@@ -65,6 +65,9 @@ ES,Bilbaobizi (Bilbao),"Bilbao, ES",nextbike_bo,https://www.bilbaobizi.bilbao.eu
...
+FR,Vélopop,"Avignon, FR",Vélopop_FR_Avignon",https://www.velopop.fr/,https://avignon-gbfs.klervi.net/gbfs/gbfs.json
```
Comment (bddq) [inline systems.csv]: I suggest IDs for Smoove networks to start with `smoove` (and also avoid diacritics).
```suggestion
FR,Vélopop,"Avignon, FR",smoove_velopop",https://www.velopop.fr/,https://avignon-gbfs.klervi.net/gbfs/gbfs.json
```

**Review by bddq — CHANGES_REQUESTED** [review]

File: systems.csv (hunk 1 above)
Comment (sthocs) [reply systems.csv:68]: Could be cool indeed, but I just copied the ID they expose in the feeds, I think it's up to them to normalize their ID like that.

**Comment by sthocs** [comment]
Thanks for the quick review! Sorry for the broken quote x_x

**Pull request merged**